frontend/src/components/Content.tsx

```diff
@@ -180,9 +180,15 @@ const Content: React.FC<ContentProps> = ({ isExpanded, showChatBot, openChatBot
     setShowAlert(false);
   };
 
-  const openGraphUrl = `https://bloom-latest.s3.eu-west-2.amazonaws.com/assets/index.html?connectURL=${
-    userCredentials?.userName
-  }@${localStorage.getItem('hostname')}%3A${localStorage.getItem('port') ?? '7687'}&search=Show+me+a+graph`;
+  const handleOpenGraphClick = () => {
+    const bloomUrl = process.env.BLOOM_URL;
+    const connectURL = `${userCredentials?.userName}@${localStorage.getItem('hostname')}%3A${
```

Comment:
misses the `neo4j+s://` prefix ??

Reply:
It does not require the protocol according to what you shared here [neo4j%2B://<username>@<database-host>%3A<port>]. Also on implementation it redirects properly.